packages/web-app/src/validation/invalid-appeal-details.js

```diff
@@ -0,0 +1,18 @@
+const { body } = require('express-validator');
+
+const invalidAppealDetails = () =>
+  body('invalid-appeal-reasons')
+    .notEmpty()
+    .withMessage('Select why the appeal is invalid')
+    .bail()
+    .custom((value, { req }) => {
+      if (value.includes('other')) {
```

Comment:
The `else` for this conditional isn't covered by the unit tests, which means we should either add a test to show that this validator passes with a value other than `other` or add an `istanbul ignore else` rule to exclude this from the coverage.

Reply:
Looks like this one still needs to be done.